cl/favorites/models.py

```diff
@@ -83,6 +92,10 @@ class Meta:
         unique_together = (("docket", "tag"),)
 
 
+@pghistory.track(
+    pghistory.Snapshot(),
+    obj_field=pghistory.ObjForeignKey(related_name="history"),
+)
 class UserTag(AbstractDateTimeModel):
```

Comment:
We should ignore view_counts, to save ourselves an insert for every page load.

Reply:
We can exclude the field, the problem is that we won't be able to revert the object, according to the documentation if we exclude fields and try to revert an object it will throw an RuntimeError exception: https://django-pghistory.readthedocs.io/en/2.5.1/reversion.html#reverting-objects

Reply:
That's OK if we can't revert easily. I think that's preferred vs having the history table get really big or incur a performance penalty when a page becomes viral.

Reply:
Did we exclude view count somewhere that I didn't see?

Reply:
I missed the last message, i just pushed this change.